### PR TITLE
feat(hanko-elements): add light dom support

### PR DESCRIPTION
**Pull request by bjoern-m**

### CODE REVIEW

File: examples/vue/package.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
+{
+  "name": "vue",
+  "scripts": {
+    "start": "vite --port 3000"
+  },
+  "dependencies": {
+    "@teamhanko/hanko-elements": "^0.0.7-alpha",
+    "vue": "^3.2.38",
```
[the remark this answers is not in the document]
Comment (bjoern-m) [reply examples/vue/package.json:8]: @sonatype-lift ignoreall

File: examples/angular/package.json [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  "name": "example-angular",
+  "private": true,
+  "scripts": {
+    "start": "ng serve --port 3000"
+  },
+  "dependencies": {
+    "@angular/animations": "^14.2.0",
+    "@angular/common": "^14.2.0",
+    "@angular/compiler": "^14.2.0",
+    "@angular/core": "^14.2.0",
```
[the remark this answers is not in the document]
Comment (bjoern-m) [reply examples/angular/package.json:11]: @sonatype-lift ignoreall

File: examples/express/package.json [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+  "name": "example-express",
+  "private": true,
+  "scripts": {
+    "start": "node src/server.js"
+  },
+  "dependencies": {
+    "cookie-parser": "^1.4.6",
+    "cors": "^2.8.5",
+    "dotenv": "^16.0.2",
+    "express": "^4.18.1",
```
[the remark this answers is not in the document]
Comment (bjoern-m) [reply examples/express/package.json:11]: @sonatype-lift ignoreall

File: examples/angular/package.json [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+  "private": true,
+  "scripts": {
+    "start": "ng serve --port 8888",
+    "start-docker": "ng serve --host=0.0.0.0 --port 8888"
+  },
+  "dependencies": {
+    "@angular/animations": "^14.2.0",
+    "@angular/common": "^14.2.0",
+    "@angular/compiler": "^14.2.0",
+    "@angular/core": "^14.2.0",
```
[the remark this answers is not in the document]
Comment (lfleischmann) [reply examples/angular/package.json:12]: @sonatype-lift ignoreall

**Review by lfleischmann — APPROVED** [review]

**Review by aspeteRakete — APPROVED** [review]

File: examples/angular/package.json (hunk 4 above)
[the remark this answers is not in the document]
Comment (aspeteRakete) [reply examples/angular/package.json:12]: @sonatype-lift exclude examples/angular/package.json

**Pull request merged**